src/main.rs

```diff
@@ -936,6 +1010,15 @@ fn print_chips() -> Result<i32, anyhow::Error> {
     Ok(0)
 }
 
+/// Print a separator before the device messages start.
+fn print_separator() {
+    eprintln!("{}", "─".repeat(80).dimmed());
+}
+
+fn print_backtrace_start() {
+    println!("{}", "stack backtrace:".dimmed());
```

Comment:
Should this also be `eprintln!`? Or should we use `println!` 4 lines above?

Reply:
good catch. I think there's no need to print the line to stderr :D